src/mips/psyqo/src/advancedpad.cpp

```diff
@@ -109,23 +117,24 @@ uint8_t psyqo::AdvancedPad::outputMultitap(unsigned ticks) {
             break;
     }
 
-    return data_out;
+    return dataOut;
 }
 
 void psyqo::AdvancedPad::processChanges(Pad pad) {
+    const unsigned padIndex = static_cast<unsigned>(pad);
     bool padConnected = isPadConnected(pad);
-    bool wasConnected = m_connected[pad];
+    bool wasConnected = m_connected[padIndex];
     if (wasConnected && !padConnected) {
         m_callback(Event{Event::PadDisconnected, pad});
     } else if (!wasConnected && padConnected) {
         m_callback(Event{Event::PadConnected, pad});
     }
-    m_connected[pad] = padConnected;
+    m_connected[padIndex] = padConnected;
     if (!padConnected) return;
 
-    uint32_t mask = 1;
-    uint32_t padData = m_padData[pad][1];
-    uint32_t buttons = m_buttons[pad];
+    uint16_t mask = 1;
```

Comment:
Downgrading these to `uint16_t` will make for a worse codegen in the end. Was there a reason to do this?

Reply:
Ugh, I probably thought I was catching a past mistake on the data type.